README.md

```diff
@@ -1,3 +1,11 @@
 # AppHealthReport
 
-Module for installing AppHealthReport components
\ No newline at end of file
+The module installs AppHealthReport components:
+1) "Weekly App Health Report» report - the report collects error information for Reports, Imports, Modules, Rules, Notifications, and Trackor Mail, as well as their run times.
```

Comment:
I think the structure should be like this:

- What is it?
- What does it consist of?
- What does it do and what additional steps and settings are needed?

Reply:
I updated the README and made a similar structure in the form of a table:
| Component | Description | Additional Information |